en/views/helpers/form.rst

```diff
@@ -1606,36 +1606,37 @@ You can also change the templates at runtime using the ``templates()`` method::
 List of Templates
 -----------------
 
-A list of the default templates and the variables they can expect are:
-
-* ``button`` {{attrs}}, {{text}}
-* ``checkbox`` {{name}}, {{value}}, {{attrs}}
-* ``checkboxFormGroup`` {{label}}
-* ``checkboxWrapper`` {{label}}
-* ``dateWidget`` {{year}}, {{month}}, {{day}}, {{hour}}, {{minute}}, {{second}}, {{meridian}}
-* ``error`` {{content}}
-* ``errorList`` {{content}}
-* ``errorItem`` {{text}}
-* ``file`` {{name}}, {{attrs}}
-* ``formGroup`` {{label}}, {{input}}, {{error}}
-* ``formStart`` {{attrs}}
-* ``formEnd`` No variables are provided.
-* ``hiddenBlock`` {{content}}
-* ``input`` {{type}}, {{name}}, {{attrs}}
-* ``inputContainer`` {{type}}, {{required}}, {{content}}
-* ``inputContainerError`` {{type}}, {{required}}, {{content}}, {{error}}
-* ``inputSubmit`` {{type}}, {{attrs}}
-* ``label`` {{attrs}}, {{text}}, {{hidden}}, {{input}}
-* ``nestingLabel`` {{hidden}}, {{attrs}}, {{input}}, {{text}}
-* ``legend`` {{text}}
-* ``option`` {{value}}, {{attrs}}, {{text}}
-* ``optgroup`` {{label}}, {{attrs}}, {{content}}
-* ``radio`` {{name}}, {{value}}, {{attrs}}
-* ``radioWrapper``  {{input}}, {{label}}
-* ``select`` {{name}}, {{attrs}}, {{content}}
-* ``selectMultiple`` {{name}}, {{attrs}}, {{content}}
-* ``submitContainer`` {{content}}
-* ``textarea``  {{name}}, {{attrs}}, {{value}}
+A list of the default templates, their default format and the variables they expect are:
+
+* ``button`` : ``<button{{attrs}}>{{text}}</button>``
```

Comment:
Just an FYI... they were out of sync when I made this edit.  (I think there
were like two templates missing).  So no, I don't think it will be "easier"
to get of sync, because it's already easy :)




Thank you,


Did you know, it takes longer to reply to an email than to write one.
Stop the downward spiral.  Read the email charter
<http://www.emailcharter.org/>.
--
Richard Kersey


On Thu, Mar 3, 2016 at 2:38 PM, Mark Story <notifications@github.com> wrote:

> In en/views/helpers/form.rst
> <https://github.com/cakephp/docs/pull/3791#discussion_r54934965>:
>
> > -* ``inputContainerError`` {{type}}, {{required}}, {{content}}, {{error}}
> > -* ``inputSubmit`` {{type}}, {{attrs}}
> > -* ``label`` {{attrs}}, {{text}}, {{hidden}}, {{input}}
> > -* ``nestingLabel`` {{hidden}}, {{attrs}}, {{input}}, {{text}}
> > -* ``legend`` {{text}}
> > -* ``option`` {{value}}, {{attrs}}, {{text}}
> > -* ``optgroup`` {{label}}, {{attrs}}, {{content}}
> > -* ``radio`` {{name}}, {{value}}, {{attrs}}
> > -* ``radioWrapper``  {{input}}, {{label}}
> > -* ``select`` {{name}}, {{attrs}}, {{content}}
> > -* ``selectMultiple`` {{name}}, {{attrs}}, {{content}}
> > -* ``submitContainer`` {{content}}
> > -* ``textarea``  {{name}}, {{attrs}}, {{value}}
> > +A list of the default templates, their default format and the variables they expect are:
> > +
> > +* ``button`` : ``<button{{attrs}}>{{text}}</button>``
>
> Doesn't this make it much easier for the default templates and docs to get
> out of sync?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cakephp/docs/pull/3791/files#r54934965>.
>